common/lib/dependabot/errors.rb

```diff
@@ -230,6 +230,11 @@ def self.updater_error_details(error)
           message: error.message
         }
       }
+    when Dependabot::GitAuthToken
```

Comment:
seems this is a new error, thus it'll need to handled E2E

Reply:
ill raise API PR for this